runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -531,6 +531,33 @@ private void hop(boolean previous)
 		int worldIdx = worlds.indexOf(currentWorld);
 		int totalLevel = client.getTotalLevel();
 
+		boolean limitPing = true;
+
+		// Don't limit quick-hopping on ping if there are no worlds below desired ping
+		if (config.quickHopPingLimit() == 0 || worlds.stream().noneMatch(world -> getStoredPing(world) <= config.quickHopPingLimit()))
```

Comment:
I don't see any reason to have this? There is already logic to handle if it cannot find a world to hop to, which seems fine?